src/alpine/3.19/amd64/Dockerfile

```diff
@@ -37,6 +46,20 @@ RUN apk update && apk add --no-cache \
         shadow \
         sudo \
         tzdata \
+        userspace-rcu \
         util-linux-dev \
         which \
         zlib-dev
+
+RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
+    lttng-ust
+
+RUN curl -L https://github.com/PowerShell/PowerShell/releases/download/v7.4.2/powershell-7.4.2-linux-musl-x64.tar.gz -o /tmp/powershell.tar.gz
+
+RUN mkdir -p /opt/microsoft/powershell/7
+
+RUN tar zxf /tmp/powershell.tar.gz -C /opt/microsoft/powershell/7
+
+RUN chmod +x /opt/microsoft/powershell/7/pwsh
+
+RUN ln -s /opt/microsoft/powershell/7/pwsh /usr/bin/pwsh
```

Comment:
This should all be executed in one `RUN` command. There are several reasons for this:
* It's one logical group of work: installing PowerShell so it's reasonable to have the commands grouped together.
* It can produce a smaller image. Once a `RUN` command has executed, it writes a layer into the image. Those layers are cumulative. Deleting files from earlier layers don't actually make the image smaller. The file still exists in the earlier image layer and is essentially cloaked in the final image.